src/main/java/nextstep/ladder/domain/MatchedResult.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.ladder.domain;
+
+import java.util.Map;
+
+public class MatchedResult {
+    private static final String CHECK_TARGET_PLAYER = "입력하신 플레이어가 존재하지 않습니다.";
+    private static final String ALL = "all";
+
+    private final Map<String, String> matchedResult;
+
+    public MatchedResult(Map<String, String> matchedResult) {
+        this.matchedResult = matchedResult;
+    }
+
+    public boolean isAll(String target) {
+        return ALL.equals(target);
```

Comment:
```suggestion
        return Objects.equals(ALL, target);
```
이렇게 하면 null 체크 까지 할 수 있습니다!

Reply:
전체 출력 여부를 처리하는 로직은 view에게 위임하면 어떨까요?

Reply:
`Objects.equals(ALL, target);`을 통해 null까지 확인할 수 있다는 것을 처음 알게된 것 같습니다 감사합니다!! :)

처음 구현 시, 전체 출력 여부 비교 로직을 컨트롤러, 뷰 둘 중 어디에 두어야 할 지 고민을 했었습니다.
뷰에서는 최대한 처리값만 받은 후 해당 값만 단순 출력하게만 하는 것이 더 좋겠다는 생각과, 
뷰에서 비교하게 한다면, 뷰의 부담이 늘어나게 되고 또한 `all`도 플레이어 이름과 같은 개념이라고 간주했고, 이러한 도메인 관련 사항에 대한 내용을 뷰에서 알게되는 거라고 생각했기 때문에 컨트롤러에서 구현을 했었던 것 같습니다!

어떠한 이유에 있어서 출력 여부 처리 로직을 view로 위임하라고 피드백 주신건지 여쭤봐도 될까요??!

해당 사항 뷰로 위임하도록 수정하였습니다!

Reply:
`Objects.equals(ALL, target);`을 통해 null까지 확인할 수 있다는 것을 처음 알게된 것 같습니다 감사합니다!! :)

처음 구현 시, 전체 출력 여부 비교 로직을 컨트롤러, 뷰 둘 중 어디에 두어야 할 지 고민을 했었습니다.
제가 뷰에서 구현하면 좋겠다고 생각한 이유는

1. 뷰에서는 최대한 처리값만 받은 후 해당 값만 단순 출력하게만 하는 것이 좋지 않을까
  -> 뷰에서 비교하게 한다면, 뷰의 부담이 늘어난다고 생각

2. `all`도 플레이어 이름과 같은 개념이라고 간주했고, 아래와 같이 출력 여부를 처리하는 로직을 뷰에 넣으면 뷰에서 도메인에 대한 내용을 알게 되는 것이 아닐까?? 라고 생각하여 컨트롤러에서 구현을 했었던 것 같습니다!

피드백 받은 후 생각해보니, 출력 여부 비교를 위해 필요한 두 개 인자 `MatchedResult`는 데이터를 저장하고 전달하는 객체이고 나머지 하나는 플레이어 이름이긴 하지만 `String`타입이기 때문에 DTO, 값 객체를 전달하는 것과 같이 도메인과 뷰의 관계가 맺어지지 않다고 볼 수도 있겠구나라고 생각이 들어 해당 사항 뷰로 위임하도록 수정하였습니다!

어떠한 이유에 있어서 출력 여부 처리 로직을 view로 위임하라고 피드백 주신건지 여쭤봐도 될까요??!

Reply:
@seongbeenkim
all이라는 키워드를 사용하는 것 자체가 사실 출력을 위한 행위이기 때문입니다. view를 위한 행위인거죠!
반대로 생각하면 도메인에서 출력 로직 때문에 all 이라는 키워드를 알 필요가 있을까요?

사용자한테 입력받은 값을 일부만 출력할지 전체만 출력할지 결정하는 부분이고 이를 위한 인터페이스는 이미 도메인이 다 제공하고 있는 상태라서요! 이러한 이유 때문에 View에게 위임하는게 좋다고 생각했습니다 ㅎㅎ

Reply:
위 내용을 통해 어떠한 이유에서 말씀하신건지 이해가 되었습니다 
감사합니다!! :)